d3a_interface/settings_validators.py

```diff
@@ -15,75 +15,77 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
+from datetime import timedelta
+
+from pendulum import duration, Duration
+
 from d3a_interface.constants_limits import ConstSettings, GlobalConfig
 from d3a_interface.exceptions import D3ASettingsException
-from pendulum import duration, Duration
-from datetime import timedelta
 
 
 def validate_global_settings(settings_dict):
-    slot_length = settings_dict["slot_length"] \
-        if "slot_length" in settings_dict else GlobalConfig.slot_length
-    tick_length = settings_dict["tick_length"] \
-        if "tick_length" in settings_dict else GlobalConfig.tick_length
+    slot_length = (settings_dict["slot_length"]
+                   if "slot_length" in settings_dict else GlobalConfig.slot_length)
+    tick_length = (settings_dict["tick_length"]
+                   if "tick_length" in settings_dict else GlobalConfig.tick_length)
```

Comment:
```suggestion
    tick_length = getattr(settings_dict, "tick_length", GlobalConfig.tick_length)
```
Seems better to catch default values

Reply:
Sorry, my suggestion wont work here as its a dict and not a python object